### PR TITLE
Create a basic modal to prevent accidental deleting

### PR DESCRIPTION
**Pull request by felipesere**

### CODE REVIEW

**Comment by felipesere** [comment]
Should take care of #100

File: assets/css/main.scss [hunk 1]
```diff
@@ -570,3 +570,75 @@ img.dashboard_action_icon {
...
+.modal {
+  align-items: center;
+  bottom: 0;
+  display: none;
+  flex-direction: column;
+  justify-content: center;
+  left: 0;
+  overflow: hidden;
+  position: absolute;
+  position: fixed;
```
Comment (RabeaGleissner) [inline assets/css/main.scss]: this overwrites the previous line... so I think you can delete the `position: absolute`

File: assets/css/main.scss [hunk 2]
```diff
@@ -570,3 +570,75 @@ img.dashboard_action_icon {
...
+  right: 0;
+  top: 0;
+  z-index: 20;
```
Comment (RabeaGleissner) [inline assets/css/main.scss]: might be worth using variables for z-index and then importing , e.g. low = 0, medium = 10, high = 20 and highest = 30. Or even something like "background", "content-layer", "modal-layer" or something like that. 
That way it's easier to manage and you don't run into problems of people trying to outdo old z-indexes further down the line.

File: assets/css/main.scss [hunk 3]
```diff
@@ -570,3 +570,75 @@ img.dashboard_action_icon {
...
+  display: flex;
+}
+
+.modal-background {
+  background-color: rgba(10, 10, 10, .86);
+  bottom: 0;
+  left: 0;
+  position: absolute;
+  right: 0;
+  top: 0;
```
Comment (RabeaGleissner) [inline assets/css/main.scss]: I think for position: absolute you only need to specify 
- either top or bottom
- either left or right
As far as I can remember, there's no need to specify, bottom and top and left and right.

File: assets/css/main.scss [hunk 4]
```diff
@@ -570,3 +570,75 @@ img.dashboard_action_icon {
...
+  overflow: hidden;
+  margin: 0 auto;
+  width: 640px;
+  position: relative;
+}
+
+.modal-body {
+  background-color: #fff;
+  flex-grow: 1;
+  flex-shrink: 1;
```
Comment (RabeaGleissner) [inline assets/css/main.scss]: doesn't this cancel each other out? first growing and then shrinking?

File: assets/css/main.scss (hunk 1 above)
Comment (felipesere) [reply assets/css/main.scss]: 👍

File: assets/css/main.scss (hunk 3 above)
Comment (felipesere) [reply assets/css/main.scss]: I just tried it... if I drop _any_ of left/right/top/bottom the background disappears.

**Pull request merged**